lib/percy.rb

```diff
@@ -128,4 +139,8 @@ def self._keys_to_json(options)
   def self._is_debug?
     ENV['LOG_LEVEL'] == 'debug'
   end
+
+  def self._driver_has_evalaute?(driver)
+    driver.respond_to?('evaluate_script')
+  end
```

Comment:
Any reason for this being a separate method rather than just the one line in the condition?